oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleConnectorConfig.java

```diff
@@ -79,6 +79,12 @@ public String getConnectionString() {
   @Macro
   private String database;
 
+  @Name(OracleConstants.TRANSACTION_ISOLATION_LEVEL)
+  @Description("The transaction isolation level for the database session.")
+  @Macro
+  @Nullable
+  public String transactionIsolationLevel;
```

Comment:
nit: why public? it should be private and only accessible through getters.

Reply:
changed

Reply:
fixed